src/index.d.ts

```diff
@@ -6,7 +6,7 @@ export function initSha256(): Promise<void>;
  * This allows users to not have to make AS a dependency in their project.
  * @param {Uint8Array} message Message to hash
  */
-declare function sha256(message: Uint8Array): Uint8Array;
+export function sha256(message: Uint8Array): Uint8Array;
```

Comment:
Probably [this line](https://github.com/ChainSafe/as-sha256/pull/23/commits/a40f55e98c07647775fe51a4b364fe808d630eb7#diff-b739f95ace6bbf2f0fd5e8cf1dab97d8R19) unnecessary?

Reply:
I think we are using default export troughout the code so this is kinda to not break api.
It enables both `import sha256 from "@chainsafe/as-sha256"` and `import {sha256} from "@chainsafe/as-sha256"`.